client/src/app/components/history/history.directive.js

```diff
@@ -5,6 +5,17 @@
         .module('oncoscape')
         .directive('osHistory', history);
 
+    var colLabels = {
+      "ptID": "ID",
+      "study": "Study",
+      "Birth.gender": "Gender",
+      "Survival": "Survival",
+      "AgeDx": "Age of Diagnosis",
+      "TimeFirstProgression": "Time to First Progression",
+      "Status.status": "Status",
+      "Pathology.histology": "Histology",
```

Comment:
I'm not sure what are the names for these columns, but I'm not able to see them in the UI.
I'm assuming that these will be shown by #128.